tfhe-csprng/examples/generate.rs

```diff
@@ -2,15 +2,14 @@
 //! the program stdout. It can also generate a fixed number of bytes by passing a value along the
 //! optional argument `--bytes_total`. For testing purpose.
 use clap::{value_parser, Arg, Command};
-#[cfg(feature = "generator_x86_64_aesni")]
+#[cfg(all(target_arch = "x86_64", target_feature = "aes"))]
```

Comment:
also needs sse2 as far as I coud see in the block cipher ?

Reply:
rust consider that sse2 is implied by aes but I can add it for the sake of explicitness: https://doc.rust-lang.org/reference/attributes/codegen.html

Reply:
ah right, well as you wish then